spec/migrations/20181010134649_add_evm_owner_to_orchestration_stacks_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_migration
+
+describe AddEvmOwnerToOrchestrationStacks do
+  let(:orchestration_stack) { migration_stub(:OrchestrationStack) }
+
+  migration_context :up do
+    it "reset column information to avoid RangeError" do
+      expect(self).to receive(:clear_caches).at_least(:once)
```

Comment:
spec migration helper automatically resets the column information before you run migrate so this test would not fail before the fix... this change makes it fail